devicemapper/plan.sh

```diff
@@ -1,13 +1,22 @@
 pkg_name=devicemapper
 pkg_origin=core
-pkg_version="2.02.171"
+pkg_version="2.03.00"
+pkg_description="The Device-mapper is a component of the linux kernel (since version 2.6) that supports logical volume management."
+pkg_upstream_url="https://sourceware.org/lvm2/"
 pkg_dirname="LVM2.${pkg_version}"
 pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
 pkg_license=('Apache-2.0')
 pkg_source="https://mirrors.kernel.org/sourceware/lvm2/releases/LVM2.${pkg_version}.tgz"
-pkg_shasum="b815a711a2fabaa5c3dc1a4a284df0268bf0f325f0fc0f5c9530c9bbb54b9964"
-pkg_deps=(core/glibc)
-pkg_build_deps=(core/make core/gcc)
+pkg_shasum="405992bf76960e60c7219d84d5f1e22edc34422a1ea812e21b2ac3c813d0da4e"
+pkg_build_deps=(
+  core/gcc
+  core/libaio
```

Comment:
Including libaio in `pkg_deps` will automatically make it available at build time. You can safely remove this line. 😄